middleware/TestingHandler.md

```diff
@@ -14,15 +20,15 @@ Provide a component, so that application developers can test their application w
 predefined response code for each method.
 #### Random Mode Response Codes List
 
-| API Method |                              Status Code List                             |
-|:----------:|:-------------------------------------------------------------------------:|
-|     GET    | 200, 301, 307, 400, 401, 403, 404, 405, 429, 500, 502, 503, 504           |
-|    POST    | 200, 201, 204, 307, 400, 401, 403, 404, 405, 429, 500, 502, 503, 504, 507 |
-|     PUT    | 200, 201, 400, 401, 403, 404, 405, 409, 429, 500, 502, 503, 504, 507      |
-|    PATCH   | 200, 204, 400, 401, 403, 404, 405, 429, 500, 502, 503, 504                |
-|   DELETE   | 200, 204, 400, 401, 403, 404, 405, 429, 500, 502, 503, 504, 507           |
+| API Method |      Status Code List        |
+|:----------:|:----------------------------:|
+|     GET    | 429, 500, 502, 503, 504      |
+|    POST    | 429, 500, 502, 503, 504, 507 |
+|     PUT    | 429, 500, 502, 503, 504, 507 |
+|    PATCH   | 429, 500, 502, 503, 504      |
+|   DELETE   | 429, 500, 502, 503, 504, 507 |
 
 ## Future Potential Requirements
 * Support for having a default Middleware chain, which can be used as Custom Middleware chain by doing some manipulations.
-
-
+* Support for custom response bodies.
+* Conditional responses based on URL and/or header pattern matching.
```

Comment:
I think this functionality is already present [Testing Handler Samples](https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/abhsriva/Testing_Handler/docs/TestingHandlerSamples.md)